lib/nosedrum/invoker/split.ex

```diff
@@ -2,12 +2,23 @@ defmodule Nosedrum.Invoker.Split do
   @moduledoc """
   An `OptionParser.split/1`-based command processor.
 
-  This parser supports a single prefix configured via the `nosedrum.prefix`
-  configuration variable:
+  This parser supports command prefixes configured via the `nosedrum.prefix`
+  configuration variable. You can specify a single prefix (as a string):
 
       config :nosedrum,
         prefix: "!"
 
+  Or a multiple prefixes (as a list of strings):
+
+      config :nosedrum,
+        prefix: ["!", "?"]
+
+  If multiple prefixes are specified, the first match in the list will be used.
+  For example, if the prefixes are `["a", "ab"]`, then the message `"ab foo"`
+  will invoke the command `b`, with the argument `foo`.  However, if the prefixes
+  were ordered `["ab", "a"]`, the message `"ab foo"` would invoke the command
```

Comment:
Thank you for the docs, they look great!! Just a question for correctness - wouldn't the message have to be `abfoo` here?

Reply:
I think `"ab foo"` is correct, as the prefix `["a", "ab"]` would match the prefix `a` (note it will *not* match the `b`), then remove the `a`, resulting in `b foo`.

It does this because it uses the first prefix that matches in the list. In this case, it uses the prefix `a` and ignores the `ab` altogether.

Another example: the prefixes `["!", "!c"]` matching on the message `!clear` will call the command `clear`, not the command `lear`.

Reply:
Hmm, what about the reverse ordering (the second part) though, `["ab", "a"]` - won't `ab foo` look for a command named `" foo"` (note the space) then, in contrast to `abfoo` which would run `"foo"`?

Reply:
Yes, if you reversed it that would be the case.

Reply:
Hmm, do we need to change it to this way then?
```suggestion
  were ordered `["ab", "a"]`, the message `"abfoo"` would invoke the command
```